system-container/root/exports/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0",
+    "defaultValues": {
+        "OPTS": "",
+        "VAR_LIB_OPENSHIFT_INSTALLER" : "/var/lib/openshift-installer",
+        "VAR_LOG_OPENSHIFT_LOG": "/var/log/ansible.log",
+        "PLAYBOOK_FILE": "/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml",
```

Comment:
we are using only one configuration file for the system container `config.json.template` while we have two Dockerfiles where it is possible to set the current working directory, duplicate the configuration file just for this difference doesn't seem ideal.  Users of Dockerfile will need to specify this argument as the default doesn't work.

Should we bring back the symlink?  It will make the two images work in the same way.